packages/editor-ui/src/api/users.ts

```diff
@@ -146,9 +146,9 @@ export async function submitPersonalizationSurvey(
 	await makeRestApiRequest(context, 'POST', '/me/survey', params as unknown as IDataObject);
 }
 
-export async function updateRole(
+export async function updateGlobalRole(
 	context: IRestApiContext,
-	{ id, role }: { id: string; role: { scope: ScopeLevel; name: IRole } },
+	{ id, roleName }: { id: string; roleName: Exclude<IRole, 'owner'> },
```

Comment:
```ts
export type IRole = 'default' | 'owner' | 'member' | 'admin';
```

Looks like only `member` and `admin` should be allowed in this payload no?

Reply:
I don't know what `default` means here. I was going to ask. do you know know?

Reply:
Possibly the owner shell before signup. In any case, let's please restrict this to only `'member | 'admin'`.